lib/client/ipam.go

```diff
@@ -577,28 +577,26 @@ func (c ipams) RemoveIPAMHost(host *string) error {
 	return nil
 }
 
-func (c ipams) hostBlockPairs(pool common.IPNet) (map[string]string, error) {
```

Comment:
Should these be common.IPNet rather than pointers?  I thought go didn't copy structs if they weren't modified. 

I'm not really fussed either way - just trying to get more opinions on pointers vs object... :-)